package.json

```diff
@@ -19,7 +20,8 @@
     "danger": "danger",
     "fix": "eslint . --fix",
     "promote": "heroku pipelines:promote -r staging",
-    "dump-schema": "babel-node ./scripts/dump-schema.js"
+    "dump-schema": "babel-node ./scripts/dump-schema.js",
+    "heroku-postbuild": "babel index.js config.js -s inline -d build && babel lib -s inline -d build/lib && babel schema -s inline -d build/schema"
```

Comment:
interesting, in Mitosis I just do this: https://github.com/artsy/Mitosis/blob/master/package.json#L9